ods_ci/tests/Resources/CLI/ModelServing/modelmesh.resource

```diff
@@ -26,7 +26,7 @@ Verify odh-model-controller Deployment
     [Documentation]    Verifies the correct deployment of the model controller in the rhods namespace
     @{odh_model_controller} =  Oc Get    kind=Pod    namespace=${APPLICATIONS_NAMESPACE}    label_selector=control-plane=odh-model-controller    # robocop: disable
     ${containerNames} =  Create List  manager
-    Verify Deployment    ${odh_model_controller}  3  1  ${containerNames}
+    Verify Deployment    ${odh_model_controller}  1  1  ${containerNames}
```

Comment:
Is this test run only on self-managed? If not, then I'm worried about this statement:
```
If no: Set replicas to 1 in the self-managed product, but leave the managed product at 3.
```

https://issues.redhat.com/browse/RHOAIENG-7034

What are the expectations there?

Reply:
@jstourac SRE has signed off for `1`